index.css

```diff
@@ -0,0 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #f7f7f9;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.nav-bar {
+  background-color: #f7f7f9;
+  padding: 20px 15px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
```

Comment:
[OPTIONAL] you have defined the body of the page to have this font family, no need to replicate it😉 unless you are using a different font. it makes your code less professional, remember a rule of thumb in programming is that you minimize repetition (DRY😉). I think you can get rid of the font-family in other places since you already defined it for the entire body.